internal/ccapi/service_instances_test.go

```diff
@@ -111,5 +184,287 @@ var _ = Describe("GetServiceInstances", func() {
 			Expect(err).To(MatchError("error getting service instances: http response: 500"))
 		})
 	})
-
 })
+
+func fakeResponse() string {
```

Comment:
praise: thanks for adding this. It is intended to serve as documentation and helps in future developments